test/unit/loops.spec.ts

```diff
@@ -553,6 +553,7 @@ for (const testCase of [
         [tstl.LuaTarget.Lua53]: expectContinueGotoLabel,
         [tstl.LuaTarget.Lua54]: expectContinueGotoLabel,
         [tstl.LuaTarget.LuaJIT]: expectContinueGotoLabel,
+        [tstl.LuaTarget.Luau]: () => undefined, // TODO: This is N/A.
```

Comment:
Why is this this N/A? Shouldn't this be expecting the Luau continue statement?

Reply:
That test specifically looks for the label or sentinel variable generated by the code for the other targets which isn't applicable to Luau since it just uses the builtin continue without a need for either.